From 541a9cf6d6035a5f8dc47062ed665ca9f7e8b180 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Wed, 3 Jan 2018 10:38:40 +0800
Subject: [PATCH] arm: dts: rename cpu0-supply to cpu-supply for rk3288 boards

cpu0-supply is for older DTs, we had better use new name from now on.

Change-Id: Icb80f5ad4718aeb1e112f317883a0cc756a5eaef
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-android-act8846-edp.dts    | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts      | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk818-edp.dts      | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk818-hdmi.dts     | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk818-lvds.dts     | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk818-mipi-edp.dts | 2 +-
 arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts     | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-evb-android-act8846-edp.dts b/arch/arm/boot/dts/rk3288-evb-android-act8846-edp.dts
index 8302c61787ad..9485c06c638f 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-act8846-edp.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-act8846-edp.dts
@@ -130,7 +130,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &dfi {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts b/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
index 3558ce51d4dc..086312069e19 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
@@ -82,7 +82,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &cif_isp0 {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-edp.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-edp.dts
index 27a230f0035b..ccf8d6bb15e6 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-edp.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-edp.dts
@@ -81,7 +81,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &dfi {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-hdmi.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-hdmi.dts
index 3e2ca316ab2b..424201fd5cab 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-hdmi.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-hdmi.dts
@@ -81,7 +81,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &dfi {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-lvds.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-lvds.dts
index d575e5097039..cf0ecf55b534 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-lvds.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-lvds.dts
@@ -81,7 +81,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &dfi {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi-edp.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi-edp.dts
index 406aa94ed709..a433052a3fa4 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi-edp.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi-edp.dts
@@ -81,7 +81,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &gpu {
diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
index a174e62e37f2..0a90e44fe656 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk818-mipi.dts
@@ -81,7 +81,7 @@
 };
 
 &cpu0 {
-	cpu0-supply = <&vdd_cpu>;
+	cpu-supply = <&vdd_cpu>;
 };
 
 &edp_panel {
-- 
2.35.3

